content/dev-guide/_index.md

```diff
@@ -0,0 +1,21 @@
++++
```

Comment:
I think you've included some formatting for some static site generator? Otherwise I don't know what this header would be for. I would leave those out until we are actually adding a static site. Because they show up in the markdown currently.

Reply:
Yes, it is and it does.